verilog/tools/kythe/indexing_facts_tree.cc

```diff
@@ -12,12 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "verilog/tools/kythe/indexing_facts_tree.h"
-
 #include <iostream>
 
 #include "absl/strings/str_join.h"
 #include "absl/strings/substitute.h"
+#include "indexing_facts_tree.h"
```

Comment:
Please always use the absolute path from the root of the project:
```
#include "verilog/tools/kythe/indexing_facts_tree.h"
```

(also in various other places)